doc/dune-files.rst

```diff
@@ -16,6 +16,26 @@ contents of all configuration files read by Dune and looks like:
 
 Additionally, they can contains the following stanzas.
 
+.. _using:
+
+using
+-----
+
+The language of configuration files read by Dune can be extended to support
+additional stanzas (eg. ``menhir``, ``coq``, ``mdx``). This is done by adding a
+line such as:
+
+.. code:: scheme
+
+    (using <stanza> <version>)
```

Comment:
Note that it is not `<stanza>` but `<plugin>`. The two usually coincide, but it's not necessarily the case. For example, the `coq` plugin introduces stanzas such as `coq.theory` but there's no `coq` stanza.

Reply:
Thank you! I did not fully realise the difference. Fixed it in 3803c54, force pushed below.

Reply:
I don't see you change yet. It still reads:

>     (using <stanza> <version>)

Reply:
Hahaha, well, my bad. I did a nice commit amend but did not add my changes…

Reply:
OK, now it does say that we're reviewing an outdated file. Sorry about the stupid loss of time.

Reply:
Thanks!

Reply:
Thank you!